safe/definitions/extra_keywords.py

```diff
@@ -113,8 +128,46 @@
         'extent of the event.')
 }
 
+extra_keyword_earthquake_source = {
+    'key': 'earthquake_source',
+    'name': tr('Source'),
+    'description': tr(
+        'Source of the earthquake, it can be initial or post-processed '
+        '(data-informed).'),
```

Comment:
I think, outside of realtime, InaSAFE users will struggle a little bit to understand this one. Advice?

Reply:
Yeah, I understand what you mean. I also will struggle with it. But, I don't what's the alternative for this case. What do you think @lucernae ?